lib/decompress/zstd_decompress.c

```diff
@@ -862,20 +862,24 @@ size_t ZSTD_execSequenceLast7(BYTE* op,
 
 typedef enum { ZSTD_lo_isRegularOffset, ZSTD_lo_isLongOffset=1 } ZSTD_longOffset_e;
 
-
 static seq_t ZSTD_decodeSequence(seqState_t* seqState, const ZSTD_longOffset_e longOffsets)
 {
     seq_t seq;
 
     U32 const llCode = FSE_peekSymbol(&seqState->stateLL);
     U32 const mlCode = FSE_peekSymbol(&seqState->stateML);
-    U32 const ofCode = FSE_peekSymbol(&seqState->stateOffb);   /* <= maxOff, by table construction */
+    U32 const ofCode = FSE_peekSymbol(&seqState->stateOffb);   /* <= MaxOff, by table construction */
 
     U32 const llBits = LL_bits[llCode];
     U32 const mlBits = ML_bits[mlCode];
     U32 const ofBits = ofCode;
     U32 const totalBits = llBits+mlBits+ofBits;
 
+    U32 const kMaxExtraBits32 =
```

Comment:
Is that a `static const` value ?

Reply:
I'd prefer not to make it static in the function, since that might make the compiler emit the code for static initialization. It could moved to static outside the function scope though.

Reply:
That's fine as is